### PR TITLE
consul 0.5.2 (new formula)

### PR DESCRIPTION
**Pull request by markcornick**
This exists in homebrew-binary, but all the other Go stuff from
Hashicorp now builds from source and is in Homebrew mainline, hence
this formula, based on the existing packer and terraform formulae.

### CODE REVIEW

File: Library/Formula/consul.rb [hunk 1]
```diff
@@ -0,0 +1,172 @@
+require "language/go"
+
+class Consul < Formula
+  desc "a tool for service discovery, monitoring and configuration"
+  homepage "https://www.consul.io"
+
```
Comment (DomT4) [inline Library/Formula/consul.rb]: No need for the empty line here.

**Comment by DomT4** [comment]
The failure is technically unrelated to the build.

```
1 file inspected, no offenses detected
Error: 1 problem in 1 formula

==> audit problems
consul:
 * Formula name conflicts with homebrew/binary/consul
```

You may wish to tee up a PR at the Binary tap to remove the formula there though.

**Comment by markcornick** [comment]
removed stray extra line, added PR to remove from homebrew-binary

**Comment by mistydemeo** [comment]
Thank you very much! Pulled.

**Pull request closed without merging**